packages/engine-test-utils/src/presets/engine-server/utils.ts

```diff
@@ -45,6 +46,56 @@ export const setupBasic: PreSetupHookFunction = async ({
   await SCHEMA_PRESETS_V4.SCHEMA_SIMPLE.create({ vault, wsRoot });
 };
 
+/**
+ /vault1
```

Comment:
awesome doc string :)

Reply:
As a nit for myself should surround this string with <pre></pre> so quic doc renders it nicely as well.